### ISSUE TITLE
помогите с кодом

### ISSUE THREAD

**Issue by Meledict** [AlexGyver/PCdisplay#11]
Добрый день. Я в программировании полный 0, только руками могу работать). Можете помочь в переделывании кода? Мне надо убрать управление мосфетами на rgb и оставить только вентиляторы и один термометр, но при этом у меня экран без драйвера. Т.е в чем заключается главная просьба: переписать код на экран 1602 без драйвера. ЗАРАНЕЕ СПАСИБО!

**Comment by dimonDDL** [comment]
Сейчас попробую.

**Comment by dimonDDL** [comment]
Вот код для 1602.
В первых строках есть настройки там я добавил ваши функции. 
[https://drive.google.com/open?id=1jBFGKIJjKPjWQ41BYb-IeBr2Mp1ZXtfr](url)

**Comment by dimonDDL** [comment]
Еще мне бы хотелось добавить отображения частоты процессора возможно допишу и сделаю Pull request.

**Comment by Meledict** [comment]
А ещё вопрос. А оно может быть не совместимо с ардуино Уно? Я собрал так сказать "тестовый стенд", а на экране пусто (первый ряд забит пикселями)

**Comment by dimonDDL** [comment]
Возможно вы неправильно подключили екран.
LCD RS pin to digital pin 12
 * LCD Enable pin to digital pin 11
 * LCD D4 pin to digital pin 10
 * LCD D5 pin to digital pin 4
 * LCD D6 pin to digital pin 8
 * LCD D7 pin to digital pin 2
 * LCD R/W pin to ground
 * LCD VSS pin to ground
 * LCD VCC pin to 5V
 * 10K resistor:
 * ends to +5V and ground
Пришлось немного изменить номера пинов так как они были уже использованы.

**Comment by Meledict** [comment]
Нет, все правильно. Если копировать строку, где идёт присвоение значений (const int rs=12, en=11...  ...lcd(rs,en...) и вставить в "привет мир" - все работает

**Comment by dimonDDL** [comment]
Можете поробовать вставить номера контектов как в оригинале а использоавние заменить.

**Comment by Meledict** [comment]
Может проблема в том, что у меня ардуино Уно, а не нано? Что бы я не изменял, получается одно и то же. Или проблема в коде (может выводы повторяются. (Короче легче уже будет купить конвертор и подождать месяцок)

**Comment by dimonDDL** [comment]
Ладно потом попробую просто сейчас у меня нет екранов 1602 вообще а так было бы интересно  сделать версию для таких екранов

**Comment by 1271** [comment]
@Meledict - попробуйте покрутить яркость экрана. Скорее всего, проблема в этом, если все подключили правильно.

**Comment by Plesh83** [comment]
Добрый день, ребята помогите плиз. Все нормально загрузилось, но иероглифы не понятные. 
![img_20180615_092508](https://user-images.githubusercontent.com/40194315/41453642-93e22e92-7087-11e8-8de5-66226e74accf.jpg)Что делать даже не знаю, уже второй день мучаюсь.

**Comment by Plesh83** [comment]
Как писать в личку?

**Comment by 1271** [comment]
> если здесь можно
Нельзя. Прикрепите файл к сообщению (просто перетащите его на окно).
Обратите внимание, что поддерживаются файлы следующих типов: GIF, JPEG, JPG, PNG, DOCX, GZ, LOG, PDF, PPTX, TXT, XLSX, ZIP.

**Comment by iwanesko** [comment]
Только переведите в формат .ino

**Comment by KirGorXXL** [comment]
> Еще мне бы хотелось добавить отображения частоты процессора возможно допишу и сделаю Pull request.

Здравствуйте! Подскажите пожалуйста, вам удалось реализовать отображение частоты ЦП? Уже месяц над этим бьюсь и всё никак не получается.

**Comment by 1271** [comment]
@KirGorXXL ответил в #1 
По идее должно помочь
